src/protocol/thread/thread.ts

```diff
@@ -673,7 +778,9 @@ export const ThreadFront = {
       // Determine the kind of this source, or its minified version.
       let kind = info.kind;
       if (kind == "prettyPrinted") {
-        const minifiedInfo = this.sources.get(info.generatedSourceIds[0]);
+        const minifiedInfo = info.generatedSourceIds
+          ? this.sources.get(info.generatedSourceIds[0])
+          : undefined;
```

Comment:
we do `const minifiedInfo = this.sources.get(info.generatedSourceIds![0]);` in another place. Is there any reason not to do that here too?

Reply:
No, there's no reason, I now changed that place to the safer version as well. Good catch!